src/style.css

```diff
@@ -0,0 +1,21 @@
+body {
+  margin: 0;
+}
+
+.stars__star {
+  display: flex;
+  flex-direction: row;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  background-image: url("./images/star.svg");
+}
+
+.active {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
```suggestion
```
redundant
stars--n modifier on each block should be responsible for amount of active stars
something like that
```
.stars--1 .stars__star:nth-child(-n+1),
...
.stars--5 .stars__star:nth-child(-n+5) {
  background-image: url("./images/star-active.svg");
}
```